rfcs/0098-community-team.md

```diff
@@ -0,0 +1,190 @@
+---
+feature: community-team
+start-date: 2021-07-31
+author: Irenes (Irene Knapp)
+co-authors: ashkitten
+shepherd-team: (names, to be nominated and accepted by RFC steering committee)
+shepherd-leader: (name to be appointed by RFC steering committee)
+related-issues: (will contain links to implementation PRs)
+---
+
+# Summary
+[summary]: #summary
+
+Establish an official community team to make moderation decisions, model social norms, and mediate interpersonal problems.
+
+# Motivation
+[motivation]: #motivation
+
+There is not currently any official mechanism for moderation action. It's not sustainable to have to call on Graham any time there's a spammer or conflict that requires moderation, and we'd like to help the community become more self-regulating.
+
+# Goals
+[goals]: #goals
+
+* Model and enforce social norms
```

Comment:
the modeling part is an important expectation of the community team. as for your other concern, we already address it in multiple ways throughout the document

from https://github.com/IreneKnapp/rfcs/blob/community/rfcs/0098-community-team.md#authority-of-the-community-team:
> The above are some suggested remedies, but these should not be taken as prescriptive. The full space of social actions is open to team members as potential tools for resolving disputes. Team members are expected to use their authority only in ways that embody justice and that are consistent with the community's values, as formalized in the code of conduct and statement of values. Team members are expected to recognize the power dynamic that they are part of, and to be proactive and explicit in making sure community members know when they're speaking in an official capacity and when they're speaking in a personal capacity. Team members are not expected to be impartial in the sense of pretending not to have personal beliefs or goals, but are expected to hold the community's values above their own, in their official actions.
>
> The community team will hold its members accountable to these ideals.

as well as https://github.com/IreneKnapp/rfcs/blob/community/rfcs/0098-community-team.md#rules-vs-principles
> The community team will make their best effort to explain why moderation decisions are being made and to tie those decisions to the statement of values.

Reply:
I would still say it's unacceptable. It creates a situation where there's a moving target in what is considered acceptable behavior, only for the benefit of the moderation team.

Coupled with language like "fascism" and "bigotry" (which at least in US, has been diluted to mean loathsome), it's creating a situation where punishment can be handed out because behavior didn't "agree" with someone on the moderation team.

I do not want to be part of community where a moderation team is projecting a some type of cultural narrative across the entire community.